app/components/Spacer/index.js

```diff
@@ -1,7 +1,4 @@
 import React from 'react';
+import styles from './styles.scss';
 
-const Spacer = ({ style }) => (
-  <div style={Object.assign({}, { flex: '1 1 auto' }, style)} />
-);
-
-export default Spacer;
+export default () => (<div className={styles.spacer}/>);
```

Comment:
In some cases we want to pass down the style prop.

Reply:
You're right, even though we have no current cases of it being used that way.

(App.js had one case which I refactored out to the corresponding scss).

Reply:
Ok, in this case it can stay like this, thanks.